### PR TITLE
refactor: remove disablemetrics for enablemetrics

### PR DESCRIPTION
**Pull request by nerdeveloper**
Signed-off-by: nerdeveloper <odirionye@gmail.com>

### CODE REVIEW

File: cmd/chartmuseum/main.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func cliHandler(c *cli.Context) {
 		UseStatefiles:          !conf.GetBool("disablestatefiles"),
 		AllowOverwrite:         conf.GetBool("allowoverwrite"),
 		AllowForceOverwrite:    !conf.GetBool("disableforceoverwrite"),
-		EnableMetrics:          !conf.GetBool("disablemetrics"),
+		EnableMetrics:          !conf.GetBool("enablemetrics"),
```
Comment (scbizu) [inline cmd/chartmuseum/main.go:98]: Should be `conf.GetBool("enablemetrics")` ?

**Review by scbizu — CHANGES_REQUESTED** [review]

**Comment by scbizu** [comment]
BTW , what issue the PR points to ?

**Comment by nerdeveloper** [comment]
@scbizu the purpose of the PR is to disable metrics by default; this is the issue: https://github.com/helm/chartmuseum/issues/447

File: cmd/chartmuseum/main.go (hunk 1 above)
Comment (nerdeveloper) [reply cmd/chartmuseum/main.go:98]: i thought `conf.GetBool("enablemetrics")` will enable the metrics by default. we are trying to disable it

Comment (cbuto) [reply cmd/chartmuseum/main.go:98]: You can use `conf.GetBool("enablemetrics")` and default the value to `false` to achieve the same thing, it just makes things a bit cleaner.

**Comment by cbuto** [comment]
Just a note, we’ll also want to reflect this change in the [helm chart](https://github.com/chartmuseum/charts) and the [docs](https://github.com/chartmuseum/www) after it’s released. @nerdeveloper let us know if you are up for that!

File: cmd/chartmuseum/main.go (hunk 1 above)
Comment (nerdeveloper) [reply cmd/chartmuseum/main.go:98]: Gotcha!

**Comment by nerdeveloper** [comment]
Sure, I am definitely up for that! I can see on the docs: https://chartmuseum.com/docs/#prometheus-metrics

Also, I will refactor it and wait till `v0.15.0` is out cc @jdolitsky

**Comment by scbizu** [comment]
Agree with @cbuto's idea , and this issue should be related to the charts and doc PR .

File: pkg/config/vars.go [hunk 2]
```diff
@@ -94,9 +94,19 @@ var configVars = map[string]configVar{
 		Default: false,
 		CLIFlag: cli.BoolFlag{
 			Name:   "disable-metrics",
-			Usage:  "disable Prometheus metrics",
+			Usage:  "(deprecated)disable Prometheus metrics",
```
Comment (jdolitsky) [inline pkg/config/vars.go:97]: can we add a space between `(deprecated)` and `disable Prometheus metrics` ?

**Review by jdolitsky — CHANGES_REQUESTED** [review]
Thanks for tackling this.

Can you also modify the README section to describe the new behavior?
https://github.com/helm/chartmuseum#prometheus-metrics

Maybe even mention the now deprecated old flag in versions `v0.14.0` and prior

File: pkg/config/vars.go (hunk 2 above)
Comment (nerdeveloper) [reply pkg/config/vars.go:97]: Sure, I will into this today.

**Comment by nerdeveloper** [comment]
> Thanks for tackling this.
> 
> Can you also modify the README section to describe the new behavior? https://github.com/helm/chartmuseum#prometheus-metrics
> 
> Maybe even mention the now deprecated old flag in versions `v0.14.0` and prior

Definitely

**Comment by nerdeveloper** [comment]
I have update chart museum Helm Chart for the Prometheus config here: https://github.com/chartmuseum/charts/pull/38

**Comment by nerdeveloper** [comment]
OMG! Yes, it has been merged! I am so happy. thank you, team.

**Pull request merged**